doc/nrf/glossary.rst

```diff
@@ -61,6 +67,11 @@ Glossary
    Clone
       A local copy of a remote Git repository obtained using the command ``git clone``.
 
+   Cluster
```

Comment:
I wonder if we should somehow tag glossary terms with contexts they belong to (i.e. Matter and Zigbee in this case). *Cluster* can be used in other contexts as well (e.g. a cluster of servers), so it might make sense to be more precise what we're defining.

Reply:
Some are already tagged as such, so if this definition applies only to Matter and Zigbee, this should be explicitly mentioned.

Reply:
fixed